src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/DynamicDependencyAttributeAlgorithm.cs

```diff
@@ -27,7 +28,7 @@ internal class DynamicDependencyAttributeAlgorithm
     {
         public static void AddDependenciesDueToDynamicDependencyAttribute(ref DependencyList dependencies, NodeFactory factory, EcmaMethod method)
         {
-            foreach (var attribute in method.GetDecodedCustomAttributes("System.Diagnostics.CodeAnalysis", "DynamicDependencyAttribute"))
+            foreach (var attribute in method.GetDecodedCustomAttributes(DiagnosticUtilities.CodeAnalysisNamespace, DiagnosticUtilities.DynamicDependencyAttribute))
```

Comment:
This is just my personal preference and I know the linker repo prefers the other style, but since these attributes are public APIs and the names are never going to change, extracting const out of it just adds extra levels of indirection that hurts readability. The original text was shorter, and it was immediately obvious what attribute this is looking for. Now one needs to follow to the const definition to discern what this is doing.